### ISSUE TITLE
Error "Find: `./*/files': No such file or directory"

### ISSUE THREAD

**Issue by jeden25** [samos123/docker-drupal#21]
Hi, I'm trying to link my remote SQL DB to your drupal image using environment variables. 

I'm typing in this into the Windows 7 cmd (I'm using boot2docker): 
`docker run --name sackman -e DB_HOST=[redacted] -e DB_NAME=[redacted] -e DB_USER=[redacted] -e DB_PASS=[redacted]  -p 80:80 samos123/drupal`

I don't think the error is related to connecting to my DB because I get this nice message: 
`=> Skipped creation of database ad_09b445d92a5cc19 - it already exists.`
(when I type in incorrect creds, I get an authentication error)

However, After the above message, I get the following error message: 
``find: `./*/files': No such file or directory``

Then, the container immediately stops. The logs show the same messages as above. Any thoughts?

**Comment by funkyfuture** [comment]
can you please post some log-context.

**Comment by jeden25** [comment]
The logs aren't much: 
![image](https://cloud.githubusercontent.com/assets/10538827/7526316/9013cf14-f4de-11e4-8148-fa3ed815b681.png)

**Comment by funkyfuture** [comment]
argh, my tries to push a debugging-image fail. can you dl this repo, make some changes i provide, build an image locally and then test it?

please don't be offended by that remark, but i find it funny that a windows user posts a log-screenshot. this docker-windows-thing really evokes some culture clash. :-D

**Comment by jeden25** [comment]
Hahahahah! That's funny, yeah, I can do that.

**Comment by funkyfuture** [comment]
okay, it's really strange that this didn't occur before.

first try:
in `entrypoint.sh#2`:

    set -ex

in `scripts/pre-launch.d/99file_perm#9`:

    find "${x}" -type d -exec chmod ug=rwx,o= '{}' \;

**Comment by jeden25** [comment]
Thanks, I can't try right at the moment but will do tomorrow.

**Comment by jeden25** [comment]
Okay, so I've done what you suggested and here are the results:
(I don't care about showing my creds anymore. It's temporary and there's nothing in the DB anyway.)
![image](https://cloud.githubusercontent.com/assets/10538827/7538275/c40354c8-f56c-11e4-8065-699d84864071.png)

And the web page shows this: 
![image](https://cloud.githubusercontent.com/assets/10538827/7538287/d2b2b144-f56c-11e4-9bc3-a5bda63941f2.png)

**Comment by funkyfuture** [comment]
yeah, i got that windows-feeling where you automatically the shit hits the fan when you look at something.

first of all, the old problem doesn't show up this time, but another one concerning your database.

concerning the first one, to check whether this actually solved it, please remove the quotes around `${x}`, because it shouldn't make a difference and therefore not solve the issue.

concerning your database, please try it with a containered database as descibed in the documentation. if there's no problem, i'd point at your database, which we cannot support.

**Comment by jeden25** [comment]
I removed the quotes and it's the same result as my last post. I then tried adding back the quotes and changing `set -ex` back to `set -e` and again got the same result. Lastly, I tried again with nothing changed and got the same result. 

Interesting, I'm not getting the original problem even when I change nothing. Maybe because I'm now building the image on my own machine? 

Anyway, I tried it with a DB in a container, and, yeah, it worked. So like you said, it's probably the remote DB that's the problem. Regardless, thanks a bunch for your help!
